### ISSUE TITLE
[Bug]: Missing Chromium resource `IDS_AX_UNLABELED_IMAGE_ROLE_DESCRIPTION` (resource 31410)

### ISSUE THREAD

**Issue by MRayermannMSFT** [electron/electron#33890]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

18.0.3

### What operating system are you using?

Windows

### Operating System Version

Windows 10 21H2

### What arch are you using?

ia32

### Last Known Working Electron version

16.0.4

### Expected Behavior

1. Open Electron app
2. Run NVDA
3. Electron app does not crash

### Actual Behavior

1. Open Electron app
2. Run NVDA
3. Electron crashes

### Additional Information

I think this is related to https://github.com/electron/electron/issues/33633?

Stacktrace:
```
[Inline Frame] electron.exe!ui::ResourceBundle::GetLocalizedStringImpl(int resource_id) Line 1162	C++
electron.exe!ui::ResourceBundle::GetLocalizedString(int resource_id) Line 771	C++
electron.exe!l10n_util::GetStringUTF16(int message_id) Line 776	C++
electron.exe!electron::ElectronContentClient::GetLocalizedString(int message_id) Line 165	C++
electron.exe!content::BrowserAccessibility::GetLocalizedRoleDescriptionForUnlabeledImage() Line 1952	C++
electron.exe!ui::AXPlatformNodeBase::GetRoleDescriptionFromImageAnnotationStatusOrFromAttribute() Line 888	C++
electron.exe!ui::AXPlatformNodeBase::ComputeAttributes(std::__1::vector<std::__1::basic_string<wchar_t,std::__1::char_traits<wchar_t>,std::__1::allocator<wchar_t>>,std::__1::allocator<std::__1::basic_string<wchar_t,std::__1::char_traits<wchar_t>,std::__1::allocator<wchar_t>>>> * attributes) Line 1177	C++
electron.exe!ui::AXPlatformNodeWin::ComputeIA2Attributes() Line 6832	C++
electron.exe!content::BrowserAccessibilityComWin::UpdateStep1ComputeWinAttributes() Line 1467	C++
electron.exe!content::BrowserAccessibilityManagerWin::OnAtomicUpdateFinished(ui::AXTree * tree, bool root_changed, const std::__1::vector<ui::AXTreeObserver::Change,std::__1::allocator<ui::AXTreeObserver::Change>> & changes) Line 739	C++
electron.exe!ui::AXTree::Unserialize(const ui::AXTreeUpdate & update) Line 1320	C++
electron.exe!content::BrowserAccessibilityManager::Unserialize(const ui::AXTreeUpdate & tree_update) Line 225	C++
electron.exe!content::BrowserAccessibilityManager::OnAccessibilityEvents(const content::AXEventNotificationDetails & details) Line 460	C++
[Inline Frame] electron.exe!content::RenderFrameHostImpl::SendAccessibilityEventsToManager(const content::AXEventNotificationDetails & details) Line 5863	C++
electron.exe!content::RenderFrameHostImpl::HandleAXEvents(const ui::AXTreeID & tree_id, mojo::StructPtr<content::mojom::AXUpdatesAndEvents> updates_and_events, int reset_token) Line 7316	C++
electron.exe!base::internal::FunctorTraits<void (network::HasTrustTokensAnswerer::*)(const network::SuitableTrustTokenOrigin &, base::OnceCallback<void (mojo::InlinedStructPtr<network::mojom::HasTrustTokensResult>)>, network::TrustTokenStore *) __attribute__((thiscall)),void>::Invoke<void (network::HasTrustTokensAnswerer::*)(const network::SuitableTrustTokenOrigin &, base::OnceCallback<void (mojo::InlinedStructPtr<network::mojom::HasTrustTokensResult>)>, network::TrustTokenStore *) __attribute__((thiscall)),base::WeakPtr<network::HasTrustTokensAnswerer>,network::SuitableTrustTokenOrigin,base::OnceCallback<void (mojo::InlinedStructPtr<network::mojom::HasTrustTokensResult>)>,network::TrustTokenStore *>(void(network::HasTrustTokensAnswerer::*)(const network::SuitableTrustTokenOrigin &, base::OnceCallback<void (mojo::InlinedStructPtr<network::mojom::HasTrustTokensResult>)>, network::TrustTokenStore *) method, base::WeakPtr<network::HasTrustTokensAnswerer> && receiver_ptr, network::SuitableTrustTokenOrigin && args, base::OnceCallback<void (mojo::InlinedStructPtr<network::mojom::HasTrustTokensResult>)> && args, network::TrustTokenStore * && args) Line 542	C++
[Inline Frame] electron.exe!base::internal::InvokeHelper<1,void>::MakeItSo(void(content::RenderFrameHostImpl::*)(const ui::AXTreeID &, mojo::StructPtr<content::mojom::AXUpdatesAndEvents>, int) && functor, base::WeakPtr<content::RenderFrameHostImpl> && weak_ptr, ui::AXTreeID && args, mojo::StructPtr<content::mojom::AXUpdatesAndEvents> && args, int && args) Line 726	C++
[Inline Frame] electron.exe!base::internal::Invoker<base::internal::BindState<void (content::RenderFrameHostImpl::*)(const ui::AXTreeID &, mojo::StructPtr<content::mojom::AXUpdatesAndEvents>, int) __attribute__((thiscall)),base::WeakPtr<content::RenderFrameHostImpl>,ui::AXTreeID,mojo::StructPtr<content::mojom::AXUpdatesAndEvents>,int>,void ()>::RunImpl(void(content::RenderFrameHostImpl::*)(const ui::AXTreeID &, mojo::StructPtr<content::mojom::AXUpdatesAndEvents>, int) && functor, std::__1::tuple<base::WeakPtr<content::RenderFrameHostImpl>,ui::AXTreeID,mojo::StructPtr<content::mojom::AXUpdatesAndEvents>,int> && bound, std::__1::integer_sequence<unsigned int,0,1,2,3>) Line 779	C++
electron.exe!base::internal::Invoker<base::internal::BindState<void (content::RenderFrameHostImpl::*)(const ui::AXTreeID &, mojo::StructPtr<content::mojom::AXUpdatesAndEvents>, int) __attribute__((thiscall)),base::WeakPtr<content::RenderFrameHostImpl>,ui::AXTreeID,mojo::StructPtr<content::mojom::AXUpdatesAndEvents>,int>,void ()>::RunOnce(base::internal::BindStateBase * base) Line 752	C++
[Inline Frame] electron.exe!base::OnceCallback<void ()>::Run() Line 142	C++
electron.exe!base::`anonymous namespace'::PostTaskAndReplyRelay::RunTaskAndPostReply(base::`anonymous namespace'::PostTaskAndReplyRelay relay) Line 100	C++
electron.exe!base::internal::FunctorTraits<void (*)(base::(anonymous namespace)::PostTaskAndReplyRelay),void>::Invoke<void (*)(base::(anonymous namespace)::PostTaskAndReplyRelay),base::(anonymous namespace)::PostTaskAndReplyRelay>(void(*)(base::`anonymous namespace'::PostTaskAndReplyRelay) && function, base::`anonymous namespace'::PostTaskAndReplyRelay && args) Line 437	C++
[Inline Frame] electron.exe!base::OnceCallback<void ()>::Run() Line 142	C++
electron.exe!base::TaskAnnotator::RunTaskImpl(base::PendingTask & pending_task) Line 135	C++
[Inline Frame] electron.exe!base::TaskAnnotator::RunTask(perfetto::StaticString event_name, base::PendingTask & pending_task, base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::DoWorkImpl::<lambda_0> && args) Line 74	C++
[Inline Frame] electron.exe!base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::DoWorkImpl(base::sequence_manager::LazyNow * continuation_lazy_now) Line 385	C++
electron.exe!base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::DoWork() Line 290	C++
electron.exe!base::MessagePumpForUI::DoRunLoop() Line 220	C++
electron.exe!base::MessagePumpWin::Run(base::MessagePump::Delegate * delegate) Line 79	C++
electron.exe!base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::Run(bool application_tasks_allowed, base::TimeDelta timeout) Line 500	C++
electron.exe!base::RunLoop::Run(const base::Location & location) Line 141	C++
electron.exe!content::BrowserMainLoop::RunMainMessageLoop() Line 1058	C++
electron.exe!content::BrowserMainRunnerImpl::Run() Line 156	C++
electron.exe!content::BrowserMain(content::MainFunctionParams parameters) Line 30	C++
electron.exe!content::RunBrowserProcessMain(content::MainFunctionParams main_function_params, content::ContentMainDelegate * delegate) Line 639	C++
electron.exe!content::ContentMainRunnerImpl::RunBrowser(content::MainFunctionParams main_params, bool start_minimal_browser) Line 1172	C++
electron.exe!content::ContentMainRunnerImpl::Run() Line 1043	C++
electron.exe!content::RunContentProcess(content::ContentMainParams params, content::ContentMainRunner * content_main_runner) Line 405	C++
electron.exe!content::ContentMain(content::ContentMainParams params) Line 433	C++
electron.exe!wWinMain(HINSTANCE__ * instance, HINSTANCE__ *, wchar_t * cmd, int) Line 238	C++
electron.exe!FiberBinder(void * params) Line 80	C++
KernelBase.dll!_BaseFiberStart()	Unknown
KernelBase.dll!_BaseFiberStart@0()	Unknown
ntdll.dll!_RtlUserFiberStart@0()	Unknown
```

**Comment by MRayermannMSFT** [comment]
Hmmm it looks like this was supposed to be fixed in https://github.com/electron/electron/pull/33616 but I am definitely still seeing it.

**Comment by MRayermannMSFT** [comment]
Ah, on the original PR, someone else has commented saying the backport didn't work: https://github.com/electron/electron/pull/33614#issuecomment-1094348516

**Comment by MRayermannMSFT** [comment]
Self closing. Now I understand the timeline. Fix in 18.0.2 didn't work, real fix is in 18.1.0.
